portfolio/src/main/java/com/google/sps/servlets/FetchContactsServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.sps.data.Contact;
+
+
+/** Servlet responsible for listing tasks. */
+@WebServlet("/fetch-contacts")
+public class FetchContactsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("Task").setOrderBy(OrderBy.desc("name")).build();
+    QueryResults<Entity> results = datastore.run(query);
+
+    List<Contact> tasks = new ArrayList<>();
+    while (results.hasNext()) {
+      Entity entity = results.next();
+      String email = entity.getString("email");
+      String name = entity.getString("name");
+      String other = entity.getString("other");
+      Contact task = new Contact(name,email,other);
```

Comment:
There should be a space between the commas:
```suggestion
      Contact task = new Contact(name, email, other);
```

See [4.6.2](https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace) for details.